File: SustitucionMOAModel/Dto/LegajoDto.cs
using System;

namespace SustitucionMOAModel.Dto
{
    public class LegajoDto
    {        
        public int SolpId { get; set; }
        public int PeticionDeOfertaId { get; set; }        
        public DateTime Fecha { get; set; }
        public string Observacion { get; set; }
        public int? UsuarioId { get; set; }
        public UsuarioDto Usuario { get; set; }
        public int? ArchivoId { get; set; }
        public string FechaFormateado { get; set; }
        public bool Leido { get; set; } = true;
        public string Tipo { get; set; }
        public int? PeticionDeOfertaUsuarioId { get; set; }
    }
}

File: SustitucionMOA/src/app/modelos/compras/legajoDto.ts
export interface LegajoDto {
    SolpId: number
    PeticionDeOfertaId: number
    Observacion: string
    FechaFormateado: string
    Tipo: string
    Leido: boolean
    ArchivoId?: number
    Usuario: any
}

export namespace LegajoTipo {
    export const ChatExterno = "Chat Externo"
    export const ChatInterno = "Chat Interno"
    export const CierreOferta = "Cierre de Oferta"
    export const Circular = "Circular"
    export const Cotizacion = "Cotización"
    export const Legajo = "Legajo"
    export const PeticionDeOferta = "Petición de Oferta"
    export const PeticionDeOfertaAgrupada = "Petición de Oferta Agrupada"
    export const PeticionDeOfertaVisualizacionPrecio = "Visualizacion de Precio"
    export const Pliego = "Pliego"
    export const RevisionTecnica = "Revisión tecnica"
    export const Solp = "SOLP"
    export const SolpArchivos = "SOLP Archivos"
    export const HistorialMovimientos = "HistorialMovimientos"
}
